composer.json

```diff
@@ -11,10 +11,10 @@
         "jean85/pretty-package-versions": "^1.1",
         "nette/robot-loader": "^3.0",
         "nette/utils": "^2.5",
-        "nikic/php-parser": "4.0.x-dev",
+        "nikic/php-parser": "^4.0",
         "phpdocumentor/reflection-docblock": "^4.3",
         "phpdocumentor/type-resolver": "^0.4",
-        "rector/better-reflection": "^3.0.6",
+        "roave/better-reflection": "dev-master",
```

Comment:
Waiting for https://github.com/Roave/BetterReflection/pull/409#issuecomment-369405816

Reply:
Such PRs could be added here via https://blog.martinhujer.cz/17-tips-for-using-composer-efficiently/#comment-3697775212

Reply:
@TomasVotruba Thanks, didn't know that one